README.md

```diff
@@ -1,73 +1,137 @@
+  in the Tarantool documentation.
+
+### Connectors
+
+We have connectors for many popular languages:
+
+* [Python](https://github.com/igorcoding/asynctnt)
+* [PHP](https://github.com/tarantool-php/client)
+* [Java](http://github.com/tarantool/tarantool-java/)
+* [Go](https://github.com/tarantool/go-tarantool)
+
+[Check out connector details for other languages](https://www.tarantool.io/doc/2.2/book/connectors/)
+and connect to your current instance of Tarantool.
+
+## Use cases
+
+* Transactional cache
+* Persistent queues
+* ETL
+* Base for IoT platform
+
+## Ready-made examples
+
+We have a frawework for creating of distributed apps called [Tarantool Cartridge](https://github.com/tarantool/cartridge).
+See the [tarantool/examples](https://github.com/tarantool/examples) repository
+with source code of typical projects based on this framework.
+
+## Who uses Tarantool
+
+* Nokia uses Tarantool as part of their IoT platform
+  (in-memory database + Lua scripting)
+* Russian Enterprise: banks, telecoms, retails
+* Mail.Ru, a major Russian Internet company, uses Tarantool for its email
+  service, ad systems, social services, media projects
+* Mastercard
+* Avito, Russian largest classified advertisements website
+* and many other companies
+
+## Community
+
+Join our Telegram [news channel](https://t.me/tarantool_news) and chats:
+
+* in [Russian](https://teleg.run/tarantoolru)
+* in [English](https://teleg.run/tarantool)
+
+## Hacking
+
+Report bugs at [here](https://github.com/tarantool/tarantool/issues).
+
+Submit your patches as instructed
+[here](https://www.tarantool.io/en/doc/2.2/dev_guide/developer_guidelines/#how-to-submit-a-patch-for-review).
+
+Ask your questions in tarantool@googlegroups.com.
+
+Discuss development issues (architecture, features, and other implementation details) at [mailing lists](https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/).
 
-[travis-badge]: https://api.travis-ci.org/tarantool/tarantool.svg?branch=master
-[travis-url]: https://travis-ci.org/tarantool/tarantool
 [gitlab-ci-badge]: https://gitlab.com/tarantool/tarantool/badges/master/pipeline.svg
 [gitlab-ci-url]: https://gitlab.com/tarantool/tarantool/commits/master
 [coverage-badge]: https://coveralls.io/repos/github/tarantool/tarantool/badge.svg?branch=master
 [coverage-url]: https://coveralls.io/github/tarantool/tarantool?branch=master
 [groups-badge]: https://img.shields.io/badge/Google-Groups-orange.svg
 [groups-url]: https://groups.google.com/forum/#!forum/tarantool
 [telegram-badge]: https://img.shields.io/badge/Telegram-join%20chat-blue.svg
-[telegram-url]: http://telegram.me/tarantool
-[slack-badge]: https://img.shields.io/badge/Slack-join%20chat-lightgrey.svg
-[slack-url]: http://slack.tarantool.org/
-[gitter-badge]: https://badges.gitter.im/Join%20Chat.svg
-[gitter-url]: https://gitter.im/tarantool/tarantool
+[telegram-url]: https://teleg.run/tarantool
```

Comment:
Don't get this change.

Reply:
Ouch, it was explained before. But now `t.me` is not blocked anymore, so there is no need to use the third-party service.